helpers/config-loader.js

```diff
@@ -1,8 +1,9 @@
 module.exports = function(file, plugins) {
-  if (plugins.globby.sync('./config/' + file).length) {
+  if (plugins.globby.sync('../../../dev/tools/frontools/configs/' + file).length) {
```

Comment:
`this.opts.configs.projectPath` is not available inside of the helpers, that's why I didn't use it. What would your preferred solution to this be?